### ISSUE TITLE
Commander requires extra button press to launch

### ISSUE THREAD

**Issue by cmclark00** [od-contrib/commander#25]
Hello, I have recently built commander for the Anbernic RG28/35/40XX series of devices. These devices are running on aarch64 linux kernel 4.9.170. for some reason when i launch the application i have to hit another button, like down on the dpad, or A on the gamepad or whatever, to get the application to display. I thought it may be something with my launcher script, but it does the same thing if i just run the application directly. I was wondering if you had any idea what might be causing this in our environment, because I can't figure it out. Thanks for taking the time to read this.

**Comment by glebm** [comment]
No idea but I assume it's the environment. Does it happen with other SDL apps?

**Comment by cmclark00** [comment]
No, this is the only one. Weirdly it also happens with the old 32 bit
dingux commander as well when I built it, but everybother sdl app seems fine
